common/pom.xml

```diff
@@ -30,6 +30,7 @@
   <name>Netty/Common</name>
 
   <properties>
+    <javaModuleName>io.netty.util</javaModuleName>
```

Comment:
What's wrong with common?

Reply:
The module names should be derived from the main package within them. This is why I chose `util` instead of `common` and `channel` instead of `transport`.

Mark Reinhold: *Strongly recommend that all modules be named according to the reverse Internet domain-name convention. A module's name should correspond to the name of its principal exported API package, which should also follow that convention.*

http://blog.joda.org/2017/04/java-se-9-jpms-module-naming.html